cli/src/pcluster/api/openapi/openapi.yaml

```diff
@@ -5,22 +5,22 @@ info:
   version: 3.0.0
```

Comment:
do we need to generatePythonServer and import any change to classes under api/model?

Reply:
Yes -- updating the server then updating unit tests then updating integ tests are the remaining items.